public/js/reader.js

```diff
@@ -109,7 +111,38 @@ Reader.loadImages = function () {
             if (Reader.infiniteScroll) {
                 Reader.initInfiniteScrollView();
             } else {
-                $(document).on("click.imagemap_change_pages", "#Map area", Reader.handlePaginator);
+                $(window).on('wheel', function(e) {
+                    if(Reader.isFullscreen) {
+                        if (e.originalEvent.deltaY > 0) Reader.changePage(-1);
+                        else Reader.changePage(1);
+                        return false;
+                    }
+                });
+                $(document).on('click', function(event) {
+                    var img_y_offset = $('a#display img').offset()['top'] + $('a#display img').height();
```

Comment:
This needs some extra comments.

Reply:
when click screen js caculate img area space and window space

and check click position on page
and do if click position Y < img div height 
then caculate left or right on screen
so now we got a same work like map

i will add comment this action